src/uct/ib/base/ib_iface.c

```diff
@@ -81,12 +81,12 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "Number of send WQEs for which completion is requested.",
    ucs_offsetof(uct_ib_iface_config_t, tx.cq_moderation), UCS_CONFIG_TYPE_UINT},
 
-  UCT_IFACE_MPOOL_CONFIG_FIELDS("TX_", -1, 1024, "send",
+  UCT_IFACE_MPOOL_CONFIG_FIELDS("TX_", -1, 512, "send",
                                 ucs_offsetof(uct_ib_iface_config_t, tx.mp),
       "\nAttention: Setting this param with value != -1 is a dangerous thing\n"
       "in RC/DC and could cause deadlock or performance degradation."),
 
-  {"RX_QUEUE_LEN", "4096",
+  {"RX_QUEUE_LEN", "1024",
```

Comment:
You can increase the size on demand if you use SRQ. I think I implemented this in openib btl.